### PR TITLE
helm 5.4.0: cut final release

### PR DESCRIPTION
**Pull request by dimitarvdimitrov**
related to https://github.com/grafana/mimir/issues/8390

### CODE REVIEW

File: docs/sources/helm-charts/mimir-distributed/_index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ keywords:
   - Grafana Enterprise Metrics
   - Grafana metrics
 cascade:
-  MIMIR_DOCS_VERSION: "next"
+  MIMIR_DOCS_VERSION: "2.13.x"
```
Comment (dimitarvdimitrov) [inline docs/sources/helm-charts/mimir-distributed/_index.md:11]:
```suggestion
  MIMIR_DOCS_VERSION: "v2.13.x"
```

**Comment by dimitarvdimitrov** [comment]
hm, actually maybe we want to wait for GEM 2.13. Keep in draft until then

**Review by tacole02 — APPROVED** [review]

**Comment by QuentinBisson** [comment]
Hey @dimitarvdimitrov do you know when we can expect the release of 5.5.0 to be out?

**Comment by dimitarvdimitrov** [comment]
probably in the next couple of weeks; you can follow #9382

**Comment by QuentinBisson** [comment]
Thank you. Do you think it would be possible in the future to have the helm chart releases decoupled from the mimir releases without having to use weekly charts as it's done with Loki ? (I'm not sure where I should bring such topics, sorry about that)

**Comment by dimitarvdimitrov** [comment]
there's nothing stopping us from cutting new helm releases out of sync with Mimir releases. The community call is tomorrow, you can drop by and discuss it there. Otherwise a new issue might be a suitable place to start discussion. 

My 2c is that we're not super strict about when we make helm releases, so if there's a need for an earlier stable release, we can decide on a case-by-base basis.

**Comment by QuentinBisson** [comment]
Ah alright then I will ask next time for the new release. Sadly, I cannot join the community calls because I have kids duty at this time, or I would join every month :D

**Pull request merged**